src-site/usage/getting-started.md

```diff
@@ -100,6 +100,25 @@ Although the design system—even including _all_ of the components—is relativ
 <link rel="stylesheet" href="path/to/system.min.css" />
 ```
 
+## Local development
+This project can be run locally if you want to view the in-browser docs/demo.
+
+In order to do this, you'll need to have node v14 in use due to this project's dependency on node-sass.
```

Comment:
FWIW, I've used `nvm` (node version manager) to install 14 (my system has 16 by default). If we check in a `.nvmrc` file with `14` in it, any system using nvm will activate version 14 automatically.

Reply:
Also addressed in 6a14fce